packages/node_modules/@webex/internal-plugin-ediscovery/test/unit/spec/transforms.js

```diff
@@ -130,41 +130,37 @@ describe('EDiscovery Transform Tests', () => {
 
         object = {body: activity};
 
-        contentSummary = {};
-        contentSummary.spaceId = '0';
-        contentSummary.spaceName = 'spaceName';
-        contentSummary.isOneOnOne = false;
-        contentSummary.participants = [uuid, uuid, uuid];
-        contentSummary.participantDisplayNames = ['user1', 'user2', 'user3'];
-        contentSummary.onBehalfOfUser = uuid;
-        contentSummary.encryptionKeyUrl = keyUri;
+        contentContainer = {};
+        contentContainer.containerId = '0';
+        contentContainer.containerName = 'spaceName';
+        contentContainer.isOneOnOne = false;
+        contentContainer.participants = [{id: uuid, displayName: 'user1'}, {id: uuid, displayName: 'user2'}, {id: uuid, displayName: 'user3'}];
+        contentContainer.onBehalfOfUser = uuid;
+        contentContainer.encryptionKeyUrl = keyUri;
 
-        ctx.webex.internal.ediscovery = {getContentSummaryBySpaceId: sinon.stub().returns(Promise.resolve({body: contentSummary}))};
+        ctx.webex.internal.ediscovery = {getContentContainerByContainerId: sinon.stub().returns(Promise.resolve({body: contentContainer}))};
       });
 
-      afterEach(() => ctx.webex.internal.ediscovery.getContentSummaryBySpaceId.resetHistory());
+      afterEach(() => ctx.webex.internal.ediscovery.getContentContainerByContainerId.resetHistory());
 
       it('Calls the correct decrypt functions when transforming post activities', () => {
         const result = Transforms.decryptReportContent(ctx, object, reportId)
           .then(() => {
-            assert.callCount(ctx.webex.internal.encryption.decryptText, 1);
```

Comment:
I'm not a fan of this style of testing. It's too concerned with **how** something is done versus **what** it does. I frankly don't care what decryption method is called as long as the results are what I'd expect.